src/index.html

```diff
@@ -9,10 +9,61 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="mac-image"
+      />
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+      <div class="review">
+        <div class="stars">
+          <div class="star-active"></div>
+          <div class="star-active"></div>
+          <div class="star-active"></div>
+          <div class="star-active"></div>
+          <div class="star"></div>
+        </div>
+        <div class="review__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-key">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+
+      <a
+        type="link"
+        class="button"
+        href="#buy"
+        data-qa="hover"
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `type` attribute is not needed for `<a>` tags, it's used for `<button>` and `<input>` elements. You should remove it to adhere to the guidelines.